ctru-rs/examples/hello-world.rs

```diff
@@ -1,56 +1,44 @@
+extern crate ctru;
```

Comment:
nit: This isn't needed (though I do see the other examples have it from before, when the crate used Rust 2015, so I'm just commenting on the new places it's added to)